src/raster/r.object.thickness/r.object.thickness.py

```diff
@@ -308,25 +357,34 @@ def main():
     min_thickness_pixel = min_thickness / resolution
     max_thickness_pixel = max_thickness / resolution
     mean_thickness_pixel = mean_thickness / resolution
+    median_thickness_pixel = median_thickness / resolution
 
     gs.message(
-        _("Thickness in map units: min %f  max %f  mean %f")
-        % (min_thickness, max_thickness, mean_thickness)
+        _(
+            "Thickness in map units:\nminimum = {}\nmaximun = {}\nmean    = {}\nmedian  = {}\n\n"
+        ).format(min_thickness, max_thickness, mean_thickness, median_thickness)
     )
     gs.message(
-        _("Thickness in pixels: min %f  max %f  mean %f")
-        % (min_thickness_pixel, max_thickness_pixel, mean_thickness_pixel)
+        _(
+            "Thickness in pixels:\nminimum = {:.4f}\nmaximum = {:.4f}\nmean   = {:.4f}\nmedian = {:.4f}"
+        ).format(
+            min_thickness_pixel,
+            max_thickness_pixel,
+            mean_thickness_pixel,
+            median_thickness_pixel,
+        )
     )
 
     # If the maximum thickness is equal to the expected maximum size print a warning
     if max_thickness >= tsize:
-        message = _(
-            "maximum thickness %(max_thickness)f is larger or equal to "
-            "the input expected maximum size %(size)f: "
-            "the border of the largest object has not been reached, "
-            "rise the expected maximum size value."
+        gs.warning(
+            _(
+                "maximum thickness {} is larger or equal to the imput \n"
+                "expected maximum size {}: the border of the largest \n"
+                "object has not been reached, rise the expected maximum \n"
+                "size value."
```

Comment:
Here you're passing from named placeholders, to unnamed placeholders. When you have more than one placeholder, and you are asking for a translation, it's usually better to use named placeholder (both % formatting and .format() formatting have their syntax to do it) or numbered placeholders ( like {0} and {1} with .format()), as in different languages, the order of words in a sentence can be different.


But since we can assume that this new string won't be translated now, you won't see an issue as the default English string (untranslated) will be returned. 

Also, you added \n at the end of the strings, was it because it rendered better for you, (in English)?